utilities/EluvioCli.js

```diff
@@ -51,6 +52,13 @@ const CmdAccountCreate = async ({ argv }) => {
       tenantId: argv.tenant,
     });
     console.log(yaml.dump(res));
+    if (argv.file) {
+      fs.appendFile(argv.file, yaml.dump(res), err => {
+        if (err) {
+          console.error(err);
+        }
+      });
+    }
```

Comment:
We should move this code into a function - something like output({argv})
If we don't do it now with this first commit it will be harder to do later